src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -16,7 +17,7 @@
      * @param type - any class of Machine sub class. For example: Truck.class or Bulldozer.class
      * @return the list of machines
      */
-    List<Object> getAll(Class type);
+    List<T> getAll(Class type);
```

Comment:
You need to parametrize type of variable 'type'

Reply:
The method getAll(Class type) in MachineService must return List<Machine>. Don’t change parameterization here.